lib/fog/compute/google/models/server.rb

```diff
@@ -554,13 +554,19 @@ def generate_ssh_key_metadata(username, key)
 
           ssh_keys = metadata_map["ssh-keys"] || metadata_map["sshKeys"] || ""
           ssh_keys += "\n" unless ssh_keys.empty?
-          ssh_keys += "#{username}:#{key.strip}"
+          ssh_keys += "#{username}:#{format_key_for_comment(key, username)}"
 
           metadata_map["ssh-keys"] = ssh_keys
           metadata[:items] = metadata_to_item_list(metadata_map)
           metadata
         end
 
+        def format_key_for_comment(key, default_comment = "")
```

Comment:
We should make the default comment not an empty string. I suggest fog-user.

Reply:
Added `fog-user` as default_comment.